examples/directory_listing.tpl.html

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2023 Michal Pokusa
+#
+# SPDX-License-Identifier: Unlicense
```

Comment:
This can use html comments to match it's filetype
```
<!--
SPDX-FileCopyrightText: 2023 Michal Pokusa

SPDX-License-Identifier: Unlicense
-->
```

Beyond that, this looks good to me. I tested successfully on a feather esp32-s2 TFT. 

Thank you @michalpokusa

Reply:
Changed in latest version, I didn't know HTML comments are allowed there, thanks.